wetr.F90

```diff
@@ -197,44 +202,60 @@ subroutine getwetr(carma, igroup, rh, rdry, rwet, rhopdry, rhopwet, rc, h2o_mass
         end select
         
         rwet  = ((c1 * rdry**c2 / (c3 * rdry**c4 - log10(humidity))) + rdry**3)**(1._f / 3._f)
-      
+
         ! Determine the wet density from the wet radius.
         r_ratio  = (rdry / rwet)**3
         rhopwet = r_ratio * rhopdry + (1._f - r_ratio) * RHO_W
-      end if
-    end if
-    
+
+      end if ! irhswell(igroup) == I_GERBER
+
+      ! Mixed aerosol paremeterization (Pengfei Yu et al., JAMES, 2015) based on
+      ! Petters and Kreidenweis (ACP, 2007) hygroscopicity parameter kappa
+      if (irhswell(igroup) == I_KAPPA) then 
+	if (temp .le. 190._f) then
+	    rwet = rdry
+	    rhopwet = rhopdry
+	else ! temp > 190
+	    rwet = rdry * (1._f + humidity*kappa/(1._f-humidity))**(1._f/3._f)
+	    r_ratio  = (rdry / rwet)**3
```

Comment:
**3.f

Reply:
It looks like this has not been changed from **3 to **3.f

Reply:
is done